docker/distgit/Dockerfile

```diff
@@ -34,8 +34,12 @@ RUN sed -i 's/Listen 80/Listen 5001/' /etc/httpd/conf/httpd.conf
 RUN mkdir /tmp/copr-dist-git
 RUN chown copr-dist-git:packager /tmp/copr-dist-git
 
-RUN mkdir /var/lock/copr-dist-git/
-RUN chown copr-dist-git:copr-dist-git /var/lock/copr-dist-git/
+# These should be created by the RPM package and tmpfiles.d but for some reason
+# it doesn't work. Workarounding it here.
+RUN mkdir /var/lock/copr-dist-git/ /home/copr-dist-git
+RUN chown copr-dist-git:copr-dist-git \
+    /var/lock/copr-dist-git/ \
+    /home/copr-dist-git
```

Comment:
We are moving from `/home` to `/var/lib` in different commit.

Reply:
Actually, it may be a bad idea.

I was creating the `/home/copr-dist-git` directory because the very next Dockerfile line is

```
RUN echo " [user]" >> /home/copr-dist-git/.gitconfig && \
```

I just checked, and in production we have this

```
[copr-dist-git@copr-dist-git ~][PROD]$ echo $HOME
/home/copr-dist-git
```

I think we should revert some of the changes we made recently in this PR and stick to `/home/copr-dist-git`